src/TrendsOverview.svelte

```diff
@@ -1143,133 +438,10 @@
             <a href="#about" class="info-link">Learn more</a>
           </p>
         </div>
-
-        <div id="info-popup-vaccine" class="info-popup">
-          <h3 class="info-header">
-            {COVID_19_VACCINATION_TITLE}
-          </h3>
-          <p class="info-text">
-            Search interest in any aspect of COVID-19 vaccination. A scaled
-            value that you can compare across regions and times.
-          </p>
-          <p>
-            <a href="#about" class="info-link">Learn more</a>
-          </p>
-        </div>
-
-        <div id="info-popup-intent" class="info-popup">
-          <h3 class="info-header">
-            {VACCINATION_INTENT_TITLE}
-          </h3>
-          <p class="info-text">
-            Search interest in the eligibility, availability, and accessibility
-            of COVID-19 vaccines. A scaled value that you can compare across
-            regions and times.
-          </p>
-          <p>
-            <a href="#about" class="info-link">Learn more</a>
-          </p>
-        </div>
-
-        <div id="info-popup-safety" class="info-popup">
-          <h3 class="info-header">
-            {SAFETY_SIDE_EFFECTS_TITLE}
-          </h3>
-          <p class="info-text">
-            Search interest in the safety and side effects of COVID-19 vaccines.
-            A scaled value that you can compare across regions and times.
-          </p>
-          <p>
-            <a href="#about" class="info-link">Learn more</a>
-          </p>
-        </div>
+        <TimeSeries id="covid-19-vaccination" regionsByPlaceId ={regionsByPlaceId} regionalTrendsByPlaceId={trends} trendLine={(t) => {return t.trends.covid19_vaccination;}} title={COVID_19_VACCINATION_TITLE} infoText="Search interest in any aspect of COVID-19 vaccination. A scaled value that you can compare across regions and times." />
+        <TimeSeries id="vaccination-intent" regionsByPlaceId ={regionsByPlaceId} regionalTrendsByPlaceId={trends} trendLine={(t) => {return t.trends.vaccination_intent;}} title={VACCINATION_INTENT_TITLE} infoText="Search interest in the eligibility, availability, and accessibility of COVID-19 vaccines. A scaled value that you can compare across regions and times." />
```

Comment:
Nit: Regarding `infoText`, there's a way in svelte to pass this as HTML instead of a property, which is a little more idiomatic. See [slots and component composition](https://svelte.dev/tutorial/slots).

Reply:
This is much cleaner, thanks!